src/Configurations/EditorLayer/EditorLayer.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { Route, BrowserRouter, Switch } from 'react-router-dom';
+import { Paneset } from '@folio/stripes/components';
+import { EditorLayer } from './EditorLayer';
+import * as Mutations from '../../data/Configurations/mutations';
+
+const renderLayer = (create) => {
+  const handleClose = () => true;
+
+  render(
+    <QueryClientProvider client={new QueryClient()}>
+      <BrowserRouter>
+        <Paneset paneTitle="ok">
+          <Switch>
+            <Route path="/">
+              <EditorLayer configurationId="id" create={create} onClose={handleClose} />
+            </Route>
+          </Switch>
+        </Paneset>
+      </BrowserRouter>
+
+    </QueryClientProvider>,
+  );
+};
+
+describe('EditorLayer', () => {
+  it('renders EditorLayer with EDIT FORM', () => {
+    renderLayer(false);
+    expect(screen.getByText('ui-remote-storage.editForm.title')).toBeVisible();
```

Comment:
```suggestion
    expect(screen.getByRole('heading', { name: /editForm/ })).toBeVisible();
```
It's [preferable to use ByRole queries](https://testing-library.com/docs/queries/about/#priority).

And that's convenient too, there are helper tricks to do that.
- First, you can install and use [browser extension](https://testing-library.com/docs/queries/about/#browser-extension) that helps you to find optimal query.
- Second, if you use `getByRole()` with some nonexistent role or name, RTL will print you all the accessible roles in the document, so you could easily choose. Try this first, it's very cosy!